taskboot/git.py

```diff
@@ -130,3 +131,35 @@ def github_release(target, args):
         release.upload_asset(name=asset_name, path=artifact_path, label=asset_name)
 
     logger.info(f"Release available as {release.html_url}")
+
+
+def github_push(args):
+
+    # Load config from file/secret
+    config = Configuration(args)
+    assert config.has_github_auth(), "Missing GitHub authentication"
+
+    # Set remote repository
+    repo_link = "https://{}:{}@github.com/{}.git".format(
```

Comment:
Make `github.com` part of the repository argument, so the command isn't GitHub-specific.

Reply:
I think the `github.com` part shouldn't be added to the repository argument, but it should be a different argument.

Reply:
Why? It is actually part of the repository URL. What if there is a git service where the repository is something like `service.com/user/repos/myrepo`?
It's better to have the full repository URL.

Reply:
Done.